app/config/config.yml

```diff
@@ -102,3 +102,11 @@ pim_reference_data:
 
 akeneo_storage_utils:
     mapping_overrides: ~
+
+pim_elastic_search:
```

Comment:
I know this is the official configuration of the bundle, but this cannot work for behat nor a docker install (it was intended like this for integration projects).

Is suggest the following (tested on my side):

In app/config.yml:
```yaml
pim_elastic_search:
    client:
        product:
            index_name: "%pim_es_index_name%"
        config:
            host: "%pim_es_host%"
            port: "%pim_es_port%"
```
Then in `app/config/parameters.yml.dist` (so it will be on parameters.yml and parameters_test.yml):
```yaml
parameters:
    ...
    pim_es_index_name: product_index
    pim_es_host:       localhost
    pim_es_port:       9200
```
This way, a user can choose different index names for prod/dev and for behat env, if using docker he can define different DNS (host) according to the containers name, and he can also choose its port (as ES uses both 9200 and 9300, but also because the user could have two ES instance with different port output).